app/services/facility_stats_service.rb

```diff
@@ -0,0 +1,62 @@
+class FacilityStatsService
+  attr_reader :stats_by_size
+
+  def initialize(facilities:, ending_period:, rate_numerator:)
```

Comment:
Just `period` is fine here for this argument name as opposed to `ending_period` - it matches the conventions we have elsewhere also.